packages/@react-native-windows/cli/src/e2etest/projectConfig.utils.ts

```diff
@@ -15,7 +15,7 @@ export async function tryMkdir(dir: string): Promise<void> {
 export async function ensureWinUI3Project(folder: string) {
   const windowsDir = path.join(folder, 'windows');
   if (fs.existsSync(windowsDir)) {
-    await fs.promises.rm(windowsDir, {recursive: true});
+    await fs.promises.rmdir(windowsDir, {recursive: true});
```

Comment:
Lefthand API not standard in Node 14 (maybe 15?). Need to check why typings allow its usage.